README.md

```diff
@@ -0,0 +1,113 @@
+<p align="center">
+  <img style="max-width: 100%;" alt="Welcome to Create UI" src="img/project-logo.png"/>
+</p>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.1.0-blue.svg?cacheSeconds=2592000" />
+  <img src="https://img.shields.io/badge/npm-%3E%3D9.5.0-blue.svg" />
+  <img src="https://img.shields.io/badge/node-%3E%3D19.7.0-blue.svg" />
+  <a href="https://github.com/vectara/create-ui#readme" target="_blank">
+    <img alt="Documentation" src="https://img.shields.io/badge/documentation-yes-brightgreen.svg" />
+  </a>
+  <a href="https://github.com/vectara/create-ui/graphs/commit-activity" target="_blank">
+    <img alt="Maintenance" src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" />
+  </a>
+  <a href="https://twitter.com/vectara" target="_blank">
+    <img alt="Twitter: vectara" src="https://img.shields.io/twitter/follow/vectara.svg?style=social" />
+  </a>
+</p>
+
+## About
+
+Create-UI is an tool for creating user interfaces, powered by the [Vectara Platform](https://vectara.com/).
+
+For an example of what you'll be building, check out [Ask News](https://asknews.demo.vectara.com) or [LegalAid](https://legalaid.demo.vectara.com).
+
```

Comment:
WDYT of adding a screenshot of AskNews here, so people know what it will look like without having to click the link? The image itself could even be a link.